yabeda-rails.gemspec

```diff
@@ -25,7 +25,7 @@ Gem::Specification.new do |spec|
   spec.required_ruby_version = ">= 2.4"
 
   spec.add_dependency "activesupport"
-  spec.add_dependency "anyway_config", ">= 1.3", "< 3"
+  spec.add_dependency "anyway_config", ">= 1.3", "< 2.5.0"
```

Comment:
Looks like [anyway_config 2.5.0 was recently released](https://rubygems.org/gems/anyway_config/versions/2.5.0) and causes an error about a @secrets instance variable.

Reply:
Can you please open a new issue in [anyway_config repo](https://github.com/palkan/anyway_config)?

Probably [this change](https://github.com/palkan/anyway_config/commit/f609a9a310d189cd16db28dedcf7f7a0d9b509aa) is conflicting with [this code](https://github.com/palkan/anyway_config/blame/b517a63e6591fcd252dd9561a4261a3caf75e169/lib/anyway/rails/loaders/secrets.rb#L31).